web/src/styles/scss/print.scss

```diff
@@ -41,6 +42,12 @@
     min-width: 100%!important;
   }
 
+  fieldset,
+  .ds-c-label {
+    width: 100% !important;
```

Comment:
![wide](https://media.giphy.com/media/4WFi6XibeDTM51XHQi/giphy.gif)

Reply:
This is the perfect gif, good job, we can all go home now